benchmark-e2e/src/main/resources/collector-config.yml

```diff
@@ -0,0 +1,34 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:55680
+
+exporters:
+  logging:
+    loglevel: info
+
+processors:
+  batch:
+  queued_retry:
+
+extensions:
+  health_check:
+
+service:
+  extensions: health_check
+  pipelines:
+    traces:
+      processors:
+        - batch
+      receivers:
+        - otlp
+      exporters:
+        - logging
```

Comment:
can we leave out the exporter, or is there a no-op exporter? in hopes of reducing the impact of running a local collector on the benchmark

Reply:
I couldn't find no-op exporter in [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/master/exporter). Wouldn't leaving out receiver throws some weird exceptions on the terminal? Since we are not exporting any data out from collector.

Reply:
I haven't tried running collector without any exporters :) But if it throws errors when there's no exporter defined than using logging seems reasonable.